### PR TITLE
Implement the core of the backend API (login/logout)

### LINKED ISSUE #26: Write core of backend-frontend API

### PR DESCRIPTION
**Pull request by InputUsername**
- Log in by sending a POST request to `/api/login` with the username/password
- Log out by sending a POST request to `/api/logout`

Fixes #26.

### CODE REVIEW

File: pydash/pydash_web/controller/login.py [hunk 1]
```diff
@@ -2,40 +2,41 @@
...
+from werkzeug.exceptions import Unauthorized
 
-from pydash_web.forms import LoginForm
+import pydash_app.user
 
 
 def login():
     if current_user.is_authenticated:
-        return redirect(url_for("pydash_web.dashboard"))
+        result = {"message": "User already logged in"}
```
Comment (FlyingBird95) [inline pydash/pydash_web/controller/login.py]: you can also use jsonify(...)

File: pydash/pydash_web/controller/login.py [hunk 2]
```diff
@@ -2,40 +2,41 @@
...
-        return __render_login_form(login_form)
+    args = __parse_arguments()
 
-    user = pydash_app.user.authenticate(login_form.username.data,
-                                        login_form.password.data)
-    if not user:
-        flash("Wrong username or password!")
-        return __render_login_form(login_form)
+    if 'username' not in args or 'password' not in args:
+        result = {"message": "Username or password missing"}
```
Comment (FlyingBird95) [inline pydash/pydash_web/controller/login.py]: same for this

File: pydash/pydash_web/routes.py [hunk 3]
```diff
@@ -17,12 +15,12 @@ def serve_react():
     return bp.send_static_file("index.html")
 
 
-@bp.route("/login", methods=["GET", "POST"])
+@bp.route("/api/login", methods=["POST"])
```
Comment (FlyingBird95) [inline pydash/pydash_web/routes.py]: you can also make an API-blueprint

File: pydash/pydash_web/controller/login.py (hunk 1 above)
Comment (InputUsername) [reply pydash/pydash_web/controller/login.py]: fixed

File: pydash/pydash_web/controller/login.py (hunk 2 above)
Comment (InputUsername) [reply pydash/pydash_web/controller/login.py]: fixed

**Pull request merged**